jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,6 +1,9 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
```

Comment:
I don't know what needs to be done for darks. I looked at one dark file and tried to include keywords that I saw. But mostly the purpose was to test the new schema on FITS files. This and all other schemas need to be reworked by step developers who know what keywords are actually needed.